### ISSUE TITLE
Can't Save running config on routers

### ISSUE THREAD

**Issue by gatuno** [imunes/imunes#57]
Hi

I'm trying to save the running configuration that I made on my routers but it doesn't work. For example, I create 3 routers, setup ospf areas using vtysh terminal, and tried to save the configuration using "copy running-config startup-config" and nothing. I also tried with the context menu on the router and Settings -> "Import running config".

I need to save my ospf modifications on the routers, if I'm doing it wrong, please tellme

Thanks

**Comment by denisSal** [comment]
Hello.

I don't know why vtysh won't save or copy the running configuration, this probably has something with the permissions or quagga/vtysh config. I will look into it next week.

I found a bug in `Import running configuration` on Linux while exploring this, so I fixed it in 88e0dfe. This will only save MAC, MTU and IPv4/v6 addresses on the interfaces, so if you need some others parameters for your configuration, you will need to enable and modify `Custom startup config`. Just click `Editor`, enter any name and click `Create`. Then, `Fill defaults` and modify the configuration as needed.

I hope that helped you.

**Comment by denisSal** [comment]
I revisited this old issue and found out running `copy running-config startup-config` on _vtysh_ only works when using `Custom startup config`.

Exploring further, I found out that our virtual filesystem has `/usr/local/etc/quagga/Quagga.conf` linking to `/boot.conf` and when _quagga_ starts with default configuration from IMUNES, there are some permission issues and _vtysh_ can't write to this file. Unlinking `/usr/local/etc/quagga/Quagga.conf` in the virtual filesystem (`/var/imunes/vroot`) and running the experiment enabled me to use the above mentioned _vtysh_ command without using `Custom startup config`.

I am not yet sure would changing this have any consequences, so I will leave the link as it is, but let this comment serve as a reminder for this issue.
